<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Orders</class>
 <widget class="QWidget" name="Form_Orders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orders</string>
  </property>
  <widget class="QGroupBox" name="groupBox_CurrentOrders">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Current Orders</string>
   </property>
   <widget class="QTableView" name="tableView_CurrentOrders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_CancelOrder">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_HistoricalOrders">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>761</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Historical Orders</string>
   </property>
   <widget class="QTableView" name="tableView_HistoricalOrders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>741</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Hide">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
